GeneratorSchema/Schema.cs

```diff
@@ -5,7 +5,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
-namespace GeneratorLib
+namespace KhronosGroup.Gltf.Generator.Schema
```

Comment:
Yeah, this change was just to break all ties between the two components and then I made it nice and kept it. I'll change the other ones.

Reply:
```diff
-namespace Generator
+namespace KhronosGroup.Gltf.Generator
-namespace GeneratorLib
+namespace KhronosGroup.Gltf.Generator
-namespace KhronosGroup.Gltf.Generator.Schema
+namespace KhronosGroup.Gltf.Generator.JsonSchema
-namespace GeneratorUnitTests
+namespace KhronosGroup.Gltf.Generator.UnitTests
glTFLoader.Shared.ArrayConverter : JsonConverter
```
Rename to `JsonSchema` because there is a class called `Schema` and it clashes from the other namespaces.
Can't move `ArrayConverter` easily since the codegen uses the actual namespace in the final code.